### PR TITLE
Minor fixes to ActivityManager

### PR DESCRIPTION
**Pull request by barronpm**
**Changes**
Uses synchronous Add() and asynchronous context creation.

### CODE REVIEW

File: Jellyfin.Server.Implementations/Activity/ActivityManager.cs [hunk 1]
```diff
@@ -40,8 +40,9 @@ public void Create(ActivityLog entry)
         /// <inheritdoc/>
         public async Task CreateAsync(ActivityLog entry)
         {
-            using var dbContext = _provider.CreateContext();
-            await dbContext.ActivityLogs.AddAsync(entry);
+            await using var dbContext = _provider.CreateContext();
```
Comment (cvium) [inline Jellyfin.Server.Implementations/Activity/ActivityManager.cs:43]: @Bond-009 What about `ConfigureAwait` here?

Comment (Bond-009) [reply Jellyfin.Server.Implementations/Activity/ActivityManager.cs:43]: Good point.
Should be done like this:
```cs
var dbContext = _provider.CreateContext();
await using dbContext.ConfigureAwait(false);
```

Comment (barronpm) [reply Jellyfin.Server.Implementations/Activity/ActivityManager.cs:43]: That doesn't compile

Comment (Bond-009) [reply Jellyfin.Server.Implementations/Activity/ActivityManager.cs:43]: IK, blame microsoft
https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-disposeasync#stacked-usings

**Pull request merged**